tgui/packages/tgui/interfaces/Atm/types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author DisturbHerb (https://github.com/DisturbHerb/)
+ * @license MIT
+ */
+
+export interface AtmData {
+  accountName: string;
+  accountBalance: number;
+	cardname: string;
```

Comment:
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
```suggestion
  cardname: string;
```